internal/component/loki/source/api/internal/lokipush/push_api_server.go

```diff
@@ -190,6 +209,11 @@ func (s *PushAPIServer) handleLoki(w http.ResponseWriter, r *http.Request) {
 			filtered[model.LabelName(processed[i].Name)] = model.LabelValue(processed[i].Value)
 		}
 
+		// Add tenant ID to the filtered labels if it is set
+		if userID != "" {
```

Comment:
nit: Is tenantID more specific than userID for variable name?

Reply:
I'm not sure :D

The variable comes from here https://github.com/grafana/alloy/blob/e3788ad454fd5c719b8cc64eeadebf6608e9641e/internal/component/loki/source/api/internal/lokipush/push_api_server.go#L159 and in the push.ParseRequest it's named userID.

This is something that is in loki as well, maybe that's where the confusion should be cleared up :D

Reply:
I can rename it for sure though :)

Reply:
Renamed to tenantID as it makes more sense in that context :)